### PR TITLE
New blog post

### PR DESCRIPTION
**Pull request by stevemac007**
This post aims to introduce 4 key devops values that I have been collecting.

Want to make sure we are all happy with these values.

### CODE REVIEW

File: _posts/2017-03-12-four-values-of-devops.markdown [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    Just like the agile transformations of the past, there is a difference between ‘Doing Agile’, and ‘Being Agile’.  ‘We do standups’ - therefore we are Agile. With the increase in adoption of devops practices, are we 'Doing devops' or are we 'Being devops'?
+    <br/>
+    <br/>
+    In this post we look into the four values that support a successful devops transformation.
+
+---
+
+A successful devops transformation sees a change in organisational culture. These changes often come in the way of adoption of specific tools or practices.
+
+To change culture, you need something more fundamental than just the introduction of new tools, or pushing everyone into Scrum teams.
```
Comment (robert-rocco) [inline _posts/2017-03-12-four-values-of-devops.markdown]: However, to change culture, you need something more fundamental than just the introduction of new tools, or pushing everyone into Scrum teams.

File: _posts/2017-03-12-four-values-of-devops.markdown [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+In a classic operating model for any change to occur multiple teams, if not departments, need to come together to make things happen. As organisations grow an even larger number of teams are required, until you reach a point where everyone is very busy, but little value is actually being delivered.
+
+No one aimed to build a process that didn’t work, these processes were evolved as a way to encode observed good practice. Their objective was to ensure quality through process. But blindly following a process misses the why. Too often organisations forget why they put these process in place, and start 'cargo culting’ their delivery approach.
+
+Are we going to fall into the same trap? Are we going to define a set of best practices and approaches, label things and create a cargo cult following of our own?
+
+How do we learn from the failures of the past, especially when learning from failure is one of the key values?
+
+We do this through understanding why these values are good for us, and we create a culture where things are regularly and vigorously challenged.  While we are constantly striving to deliver value we also need to also stay focused on ‘Why we are doing it.’
```
Comment (robert-rocco) [inline _posts/2017-03-12-four-values-of-devops.markdown]: We do this through understanding why these values are good for us, and we create a culture where things are regularly and vigorously challenged.  While we are constantly striving to deliver value we also need to also stay focused on ‘Why we are doing it.’

File: _posts/2017-03-12-four-values-of-devops.markdown [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+## Work lean
+There are many great resources on the internet that talk to the values of a Lean approach, so we won’t recreate them here.  
+
+Devops embraces the values of a lean approach as they are focused on doing enough to deliver value, no more, no less.  
+
+**“Why work lean”** - At its heart, a Lean approach aims to increase value through the reduction of waste. One of the strongest ways to reduce waste is to challenge why we do that activity in the first place.
+
+Forming small cross-functional teams with a lean approach is often one of the first steps on a devops adoption path.  These teams are empowered to deliver change in the way they see provides value to their customers, while still being accountable to they meets the goals of the broader organisation.
+
+Ensuring these teams are supported to make changes to reduce waste within the team, and provided a context where they have everything they need to succeed within the team will allow them to challenge the current blockers to delivering value.
```
Comment (robert-rocco) [inline _posts/2017-03-12-four-values-of-devops.markdown]: Ensuring these teams are supported to make changes to reduce waste within the team, and provided a context where they have everything they need to succeed will allow them to challenge the current blockers to delivering value.

File: _posts/2017-03-12-four-values-of-devops.markdown [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+## Embrace failure
+Nothing is perfect, and failure is always just around the corner.  The only way to guarantee that nothing can fail is to do nothing at all.
+
+While there are some industries that need to take a risk-averse position due to the nature of their work or the impact to society if they fail, most do not.  In fact the reverse is true.
+
+In a world where competition is increasing and the needs of customers are only becoming more demanding, the rate of change required to succeed far out-strips the current risk management processes.  In this context Failure is inevitable.
+
+**“Why embrace failure”** - Core to a devops culture is one where we recognize that things will fail. What we need to recognise is it is not the failure by which we will be measured but by how quickly and effectively we respond to the failure
+
+By creating an environment that acknowledges failure will occur, we can set ourselves to reduce the impact, and more importantly manage the splash zone of the failure.
```
Comment (robert-rocco) [inline _posts/2017-03-12-four-values-of-devops.markdown]: By creating an environment that acknowledges failures will occur, we can prepare ourselves to not only reduce the impact, but more importantly manage the splash zone of the failure.

File: _posts/2017-03-12-four-values-of-devops.markdown [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+We aim to fail before the customer sees it.  We peer-review our changes, are constantly integrating and testing our systems and deliver change through automation to increase our confidence.
```
Comment (robert-rocco) [inline _posts/2017-03-12-four-values-of-devops.markdown]: We aim to fail before the customer sees it.  We peer-review our changes, are constantly integrating and testing our systems, and deliver change through automation to increase our confidence.

File: _posts/2017-03-12-four-values-of-devops.markdown [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+If things do slip through and our failures become more public, we work with a small, focused team that have all the knowledge, skills and access to manage the incident and we constantly ask ‘why did this happen?’.  What could have we done to catch this issue earlier? What are we going to do to make sure this failure does not happen again?
```
Comment (robert-rocco) [inline _posts/2017-03-12-four-values-of-devops.markdown]: If things do slip through and our failures become more public, we work with a small, focused team that have all the knowledge, skills and access to manage the incident and we constantly ask ‘why did this happen?’.  What could we have done to catch this issue earlier? What are we going to do to make sure this failure does not happen again?

File: _posts/2017-03-12-four-values-of-devops.markdown [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+
+**“Why diversity”** - The right balance of conflict is important to ensure the team is consistently reviewing the current approach and looking for opportunities to reduce waste and deliver more value.
+
+By definition devops is the bringing together of Developers and Operations staff, that alone is more diversity than some teams have had for decades.
+
+Ensuring there is a voice for all aspects of the system involved during the planning, delivery and support of new features will only increase the chance that the right solution will be developed in the right timeframe for the right customers.
+
+## Driving a culture change
+
+You can see from these four values you can see that there is a depth to each, but combined together they form a strong force for change.
```
Comment (robert-rocco) [inline _posts/2017-03-12-four-values-of-devops.markdown]: You can see from these four values that there is a depth to each, but combined together they form a strong force for change.

File: _posts/2017-03-12-four-values-of-devops.markdown (hunk 2 above)
Comment (stevemac007) [inline _posts/2017-03-12-four-values-of-devops.markdown]: Whats the change here?

File: _posts/2017-03-12-four-values-of-devops.markdown (hunk 1 above)
Comment (stevemac007) [inline _posts/2017-03-12-four-values-of-devops.markdown]: +1

File: _posts/2017-03-12-four-values-of-devops.markdown (hunk 6 above)
Comment (stevemac007) [inline _posts/2017-03-12-four-values-of-devops.markdown]: Not sure what the change was, but did feel it read poorly, have fixed the language.

File: _posts/2017-03-12-four-values-of-devops.markdown (hunk 2 above)
Comment (stevemac007) [inline _posts/2017-03-12-four-values-of-devops.markdown]: missed the double also.

**Pull request merged**